fed/mosaic-environment/src/main/resources/CEnvironmentScheme.json

```diff
@@ -1,94 +1,165 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "CEnvironmentScheme",
+    "description": "Schema describing the JSON file structure for the environment configuration. It is is used to define which events are evaluated by the simulator. Entities entering the areas of the events, are notified by the 'EnvironmentSensorUpdates' interaction",
     "type": "object",
     "properties": {
         "events": {
+            "description": "List of events.",
             "type": "array",
-            "items": [ { "$ref": "#/definitions/event" } ]
+            "items": { "$ref": "#/definitions/event" }
         }
     },
     "definitions": {
         "event": {
+            "title": "event",
+            "description": "Object to define a single event configuration.",
             "type": "object",
             "properties": {
-                "type": { "$ref": "#/definitions/eventType" },
-                "location": { "$ref": "#/definitions/eventLocation" },
-                "time": { "$ref": "#/definitions/eventTime" }
+                "type": { 
+                    "description": "Type of the event.",
+                    "$ref": "#/definitions/eventType" 
+                },
+                "location": { 
+                    "description": "The location of the event,, this can either be a geoArea or a string representing a specific road segment.",
+                    "$ref": "#/definitions/eventLocation" 
+                },
+                "time": { 
+                    "description": "Time of the event.",
+                    "$ref": "#/definitions/eventTime" 
+                }
             },
             "required": [ "type", "location", "time" ]
         },
         "eventType": {
+            "title": "eventType",
+            "description": "Object to define the type of an event.",
             "type": "object",
             "properties": {
-                "sensorType": { "type": "string" },
-                "value": { "type": "integer", "default": 1 }
+                "sensorType": { 
+                    "description": "Represents the type of sensor value this event is emitting (e.g. Ice, Snow, or an arbitrary Obstacle)",
```

Comment:
Is the user free to name the event how he wants or is there a pre-defined list? Please comment this here in each case.

Reply:
Added "enum": [ "ICE", "SNOW", "OBSTACLE" ]
Other types are not possible?

Reply:
Just found out that there are also other types used. But it seems like it does not make sense to define an enunm, so instead I'd just extend the description.

Reply:
Just found out that there are also other types used. It seems like it does not make sense to define an enunm, so I'd just leave it in the description.

Reply:
No, we don't need an enum here. But you could extend the description saying that the name of the sensor type can be chosen freely.